frontend/src/components/NavBar.vue

```diff
@@ -1,188 +1,183 @@
+        <div class="navbar-start">
+          <a class="navbar-item" @click.prevent="newList()">
+            <span class="icon is-large navbar-icon">
+              <i class="fas fa-lg fa-plus-circle"></i>
+            </span>
+            New
+          </a>
+          <router-link v-if="currentUser.loggedIn" class="navbar-item" to="/s/user">
+            <span class="icon is-large navbar-icon">
+              <i class="fas fa-lg fa-user-circle"></i>
+            </span>
+            My Lists
+          </router-link>
+          <a href="https://aka.ms/theurlist" class="navbar-item" target="_new">
+            <span class="icon is-large navbar-icon">
+              <i class="fas fa-lg fa-question-circle"></i>
+            </span>
+            About
+          </a>
+        </div>
+
+        <div class="navbar-end">
+          <div class="navbar-item has-dropdown is-hoverable" v-if="currentUser.loggedIn">
+            <a class="navbar-link">
+              <div class="columns is-gapless is-mobile">
+                <div class="column is-narrow">
+                  <figure id="profileImage" class="image">
+                    <img class="is-rounded" :src="currentUser.profileImage" alt />
+                  </figure>
+                </div>
+                <div class="column">
+                  <span>{{ currentUser.name }}</span>
+                </div>
+              </div>
+            </a>
+            <div class="navbar-dropdown">
+              <a class="navbar-item" @click.prevent="logout()">
+                <span class="icon is-medium navbar-icon">
+                  <i class="fas fa-sign-out-alt"></i>
+                </span>
+                Log Out
+              </a>
+            </div>
+          </div>
+          <a v-else class="navbar-item" @click.prevent="login()">
+            <span class="icon is-large navbar-icon">
+              <i class="fas fa-lg fa-sign-in-alt"></i>
+            </span>
+            Login
+          </a>
+        </div>
+      </div>
+    </nav>
+  </header>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import ProgressBar from "@/components/ProgressBar.vue";
+import ModalLogin from "@/components/ModalLogin.vue";
+import UserMenu from "@/components/UserMenu.vue";
+import { UserManager } from "oidc-client";
+import User from "@/models/User";
+import config from "@/config";
+
+@Component({
+  components: {
+    ProgressBar,
+    UserMenu,
+    ModalLogin
+  }
+})
+export default class extends Vue {
+  showMenu: boolean = false;
+  showLoginModal: boolean = false;
+  userManager: UserManager;
+
+  constructor() {
+    super();
+    this.userManager = new UserManager({
+      loadUserInfo: false,
+      client_id: "b60e06af-abfe-4fbc-a762-52cd6800b985",
+      prompt: "login",
+      authority:
+        "https://testprodoh.b2clogin.com/testprodoh.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_SignUp_SignIn",
```

Comment:
I guessing there isn't some super generic endpoint we can hit? If not, let's make this also a configuration parameter.

Reply:
Moved into config.